static-analysis/teammates-eslint.yml

```diff
@@ -15,4 +15,5 @@ rules:
     no-continue: "off"
     no-mixed-operators: "off"
     no-param-reassign: ["error", { "props": false }]
+    import/no-extraneous-dependencies: "off"
```

Comment:
Can you add an empty line so `eslint-plugin-import` rules are separated from vanilla ESLint's rules?